src/metabase/driver.clj

```diff
@@ -447,7 +447,8 @@
              [clojure.lang.IPersistentVector :type/Array]
              [org.bson.types.ObjectId        :type/MongoBSONID]
              [org.postgresql.util.PGobject   :type/*]
-             [nil                            :type/*]]) ; all-NULL columns in DBs like Mongo w/o explicit types
+             [nil                            :type/*] ; all-NULL columns in DBs like Mongo w/o explicit types
+             [org.bson.types.Decimal128      :type/Decimal]]) ; Decimal128 support added, sum, avg math exp are now working on frontend for Decimal128 data type ( Mongo )
```

Comment:
This isn't the right place to put this sort of stuff, since the `org.bson` package isn't loaded unless the MongoDB driver is loaded

Take a look at `modules/drivers/mongo/src/metabase/driver/mongo.clj`